ideascube/models.py

```diff
@@ -55,6 +55,19 @@ def create_superuser(self, serial, password, **extra):
         user.save(using=self._db)
         return user
 
+    def get_queryset(self):
+        return super().get_queryset().exclude(serial='__system__')
+
+    def get_queryset_unfiltered(self):
+        return super().get_queryset()
+
+    def all(self, really=False):
```

Comment:
I had used `really` as in « yes I **really** want all of them, even the ones we normally would filter ».

Thinking about it some more now, we're probably never going to hide other users.

In addition, if we went the way you suggest and decided to hide more users in the future, we'd have to change that kwarg in favour of something expressing there are other users hidden. Doing that would be trivial, as a quick `grep include_system_user` would return all the occurences we'd have to change.

However, a `grep really` is sure to return lots and lots of results irrelevant in this context.

So yes, I agree with your suggestion, will change this ASAP.